management-interface.go

```diff
@@ -289,6 +292,279 @@ func NetworkHandler(w http.ResponseWriter, r *http.Request) {
 	tmpl.ExecuteTemplate(w, "network.html", state)
 }
 
+//getNetworkSSID -gets the ssid of the network associated with this id
```

Comment:
It conventional for Go docstrings to be a proper sentence starting with the identifier name. For example:
```
getNetworkSSID gets the ssid of the network associated with this id.
```